editor/webpack.config.js

```diff
@@ -281,7 +281,7 @@ const config = {
         contentBase: path.join(__dirname, 'resources'),
         watchContentBase: true, // Watch the above folder for changes too
         overlay: {
-          warnings: true,
+          warnings: false,
```

Comment:
This is because of the TS update. For some reason (a few hours were lost trying to figure this one out), TS 4.1+ results in the following warning, which we weren't able rectify: 
```
WARNING in ./node_modules/typescript/lib/typescript.js
Module not found: Error: Can't resolve 'perf_hooks' in '/home/rheese/Workspace/Utopia/utopia/editor/node_modules/typescript/lib'
 @ ./node_modules/typescript/lib/typescript.js
 @ ./src/core/workers/ts/ts-worker.ts
 @ ./src/core/workers/ts/ts.worker.ts
 ```
 
This appears to be related to [this change](https://github.com/microsoft/TypeScript/pull/40593), and maybe needs further investigation if it can't be fixed by replacing the ts-loader with babel